src/MongoDB/ReadConcern.c

```diff
@@ -101,6 +101,16 @@ PHP_METHOD(ReadConcern, getLevel)
 }
 /* }}} */
 
+/* {{{ proto array ReadConcern::bsonSerialize()
+   */
+PHP_METHOD(ReadConcern, bsonSerialize)
+{
+	const mongoc_read_concern_t *read_concern = phongo_read_concern_from_zval(getThis() TSRMLS_CC);
```

Comment:
For consistency with our other functions that take no arguments, I'd consider adding this:

```
if (zend_parse_parameters_none() == FAILURE) {
	return;
}
```

Also, did we need the following suppressions for PHP 5.x?

```
SUPPRESS_UNUSED_WARNING(return_value_ptr) SUPPRESS_UNUSED_WARNING(return_value_used)
```

If these warnings didn't trigger for you, you can always omit this change. I'll add them back if I see them later.